tasks/task01/narcissistic_numbers.py

```diff
@@ -0,0 +1,17 @@
+for k in range(1, 1001):
+    result=0
+    count=0
+    i = k
+    j = k
+    # count of digits in a number
+    while j>0:
+        count += 1
```

Comment:
A best practice is to separate code into functions. This allows the use of code many times and makes it versatile.

Now, in your case, the user cannot do anything. For example user want to check only one number or check from n to k

I suggest doing this:
- Move loop body to function 
- Move search count of tens to function
- Move check every number loop in other function